integrations/oci/objectstorage/src/main/java/io/helidon/integrations/oci/objectstorage/OciObjectStorageImpl.java

```diff
@@ -52,4 +52,10 @@
         return delegate.renameObject(request)
                 .await();
     }
+
+    @Override
+    public ApiOptionalResponse<GetObject.Response> getBucket(GetObject.Request request) {
```

Comment:
The request and response types are wrong (link in a different comment regarding the same).